middleware/varnish/detectors-varnish.tf

```
@@ -0,0 +1,116 @@
+# failed backend in varnish detection
+resource "signalfx_detector" "varnish_backend_failed" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Varnish Backend Failed"
+
+  program_text = <<-EOF
+    signal = data('varnish.backend_fail', filter=(filter('plugin', 'telegraf/varnish') and ${module.filter-tags.filter_custom}))${var.varnish_backend_failed_aggregation_function}${var.varnish_backend_failed_transformation_function}.publish('signal')
```

```suggestion
    signal = data('varnish.backend_fail', filter=filter('plugin', 'telegraf/varnish') and ${module.filter-tags.filter_custom}))${var.varnish_backend_failed_aggregation_function}${var.varnish_backend_failed_transformation_function}.publish('signal')
```

you do not need parenthesis here